charts/intel-gpu-resource-driver/templates/resource-driver.yaml

```diff
@@ -47,7 +47,6 @@ spec:
           mountPath: "/sysfs"
         securityContext:
           privileged: true
-          allowPrivilegeEscalation: false
           capabilities:
             drop: ["ALL"]
           readOnlyRootFilesystem: true
```

Comment:
While `allowPrivilegeEscalation` is no-op (always `false`) when `privileged` mode is used, according to docs: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.34/#securitycontext-v1-core

So are pretty much all the rest of these security settings, although that's not explicitly documented.

While it would be good to be able to restrict privileged mode with the other settings to just get (write) access to all host devices...  Based on look at CRI-O & containerd code, container engines actually ignore the other security context restrictions when `privileged` mode is used.